GliaWidgets/Sources/Extensions/UIFont+Extensions.swift

```diff
@@ -50,3 +50,22 @@ extension UIFont {
         return UIFont(descriptor: descriptor, size: size)
     }
 }
+
+extension UIFont {
+    static func scaledFont(forTextStyle: UIFont.TextStyle) -> UIFont? {
+        var descriptor = UIFontDescriptor.preferredFontDescriptor(withTextStyle: forTextStyle)
+        guard let style = FontScaling.Style(forTextStyle),
+              let description = FontScaling.theme.descriptions[style] else {
+            return nil
+        }
+        descriptor = descriptor.addingAttributes(
+            [
+                UIFontDescriptor.AttributeName.traits: [UIFontDescriptor.TraitKey.weight: description.weight]
+            ]
+        )
+        /// Create a font copy with original size to scale it with current preferred content size category
```

Comment:
Since it is a comment, not docs, we need to use two slashes.
```suggestion
        // Create a font copy with original size to scale it with current preferred content size category
```

Reply:
Fixed https://github.com/salemove/ios-sdk-widgets/commit/ed646f907a13c766eb024c9b27b5079088269d6f